code/modules/materials/Mat_Materials.dm

```diff
@@ -574,6 +574,35 @@ ABSTRACT_TYPE(/datum/material/metal)
 		setProperty("hard", 1)
 
 
+/datum/material/metal/veranium
+	mat_id = "veranium"
+	name = "veranium"
+	desc = "It looks to be sparking."
+	color = "#8effdd"
+
+	New()
+		..()
+		setProperty("electrical", 4)
+		setProperty("thermal", 4)
```

Comment:
i think setting thermal here to 1 as well would be a good hint that it has outstanding thermal properties. A little hand-wavy but "Material that stores excess thermal energy as electrical charge" might make a good lorenote/hint it for voltite.

Speaking of, may want to update the matsci in-game handbook for veranium?

```suggestion
		setProperty("thermal", 1)
```